### PR TITLE
Added tabs to show ui and script webhook

### PR DESCRIPTION
**Pull request by ngrayluna**
## Description

Adds instructions on how to troubleshoot w/ W&B App UI. Consolidates programmatic and interactive options into tab.

## Ticket

https://weightsandbiases.slack.com/archives/C01DX8LRZEJ/p1710958151018619

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [X] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [X] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [X] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

File: static/images/models/webhook_ui.png
Comment (noaleetz) [inline static/images/models/webhook_ui.png:1]: any chance we can create a gif beginning with how to get to this test page, typing some dummy words in the payload, and hitting test?

File: docs/guides/model_registry/automation.md
Comment (noaleetz) [inline docs/guides/model_registry/automation.md:1]: Can we add a note that says that you can test either when creating a webhook or when editing?

File: static/images/models/webhook_ui.png
Comment (noaleetz) [reply static/images/models/webhook_ui.png:1]: https://github.com/wandb/docodile/assets/40642416/2ad46065-ac93-4827-9349-d011a4fe3ebc

something like this

Comment (noaleetz) [reply static/images/models/webhook_ui.png:1]: we can also ask for a specific snippet from the webhook testing youtube video

Comment (ngrayluna) [reply static/images/models/webhook_ui.png:1]: @noaleetz Which do you think is more useful? Showing a .gif with a successful response or one that shows an error (e.g. like what you showed?)

Comment (ngrayluna) [reply static/images/models/webhook_ui.png:1]: For the sake of shipping faster, I'd vote for the latter. [I don't have a webhook currently configured to test/demo/get a screen recording of].

Comment (noaleetz) [reply static/images/models/webhook_ui.png:1]: Yea that works for me :)

Comment (ngrayluna) [reply static/images/models/webhook_ui.png:1]: Lol. So, I tried a 'fancy' example like what you had. But it takes W&B longer than 10 seconds to respond (which is how long I can make .gifs)

So does this dummy example suffice?

**Review by noaleetz — APPROVED** [review]
I am ok with leaving gif without only failed response for now

**Pull request merged**